draft-ietf-masque-connect-ip.md

```diff
@@ -83,7 +83,7 @@ informative:
 This document describes how to proxy IP packets in HTTP. This protocol is
 similar to UDP proxying in HTTP, but allows transmitting arbitrary IP packets.
 More specifically, this document defines a protocol that allows an HTTP client
-to create an IP tunnel through an HTTP server that acts as a proxy. This
+to create an IP tunnel through an HTTP server that acts as an IP proxy. This
```

Comment:
Amusingly, this text came from a bit of 9298 that was written by mnot. But this change LGTM.